internal/daysteps/daysteps.go

```diff
@@ -1,20 +1,74 @@
 package daysteps
 
 import (
+	"errors"
+	"fmt"
+
+	"log"
+	"strconv"
+	"strings"
 	"time"
+
+	"github.com/Yandex-Practicum/tracker/internal/spentcalories"
 )
 
 const (
-	// Длина одного шага в метрах
 	stepLength = 0.65
-	// Количество метров в одном километре
+
 	mInKm = 1000
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
-	// TODO: реализовать функцию
+	parts := strings.Split(data, ",")
+	if len(parts) != 2 {
+		return 0, 0, errors.New("incorrect data format")
+	}
+
+	steps, err := strconv.Atoi(parts[0])
+	if err != nil {
+		return 0, 0, err
```

Comment:
Лучше оборачивать ошибки
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```